tests/test_integration.py

```diff
@@ -480,3 +480,36 @@ def test_downgrade_api(self):
         ok_(response)
         eq_(response.status, 200)
         ok_(isinstance(api._encoder, JSONEncoder))
+
+@skipUnless(
+    os.environ.get('TEST_DATADOG_INTEGRATION', False),
+    'You should have a running trace agent and set TEST_DATADOG_INTEGRATION=1 env variable'
+)
+class TestRateByService(TestCase):
```

Comment:
what's the purpose of this test? `TestAPITransport` already check all of this.

Reply:
Christian anticipated a bit, that's a feature not yet available.